x/nft/keeper/keys.go

```diff
@@ -0,0 +1,66 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/internal/conv"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"github.com/cosmos/cosmos-sdk/x/nft"
+)
+
+var (
+	ClassKey             = []byte{0x01}
+	NFTKey               = []byte{0x02}
+	NFTOfClassByOwnerKey = []byte{0x03}
+	OwnerKey             = []byte{0x04}
+	ClassTotalSupply     = []byte{0x05}
+
+	Delimiter   = []byte{0x00}
+	Placeholder = []byte{0x01}
+)
+
+// StoreKey is the store key string for nft
+const StoreKey = nft.ModuleName
+
+// classStoreKey returns the byte representation of the nft class key
+func classStoreKey(classID string) []byte {
+	return append(ClassKey, []byte(classID)...)
+}
+
+// nftStoreKey returns the byte representation of the nft
+func nftStoreKey(classID string) []byte {
+	return append(append(NFTKey, []byte(classID)...), Delimiter...)
+}
+
+// classTotalSupply returns the byte representation of the ClassTotalSupply
+func classTotalSupply(classID string) []byte {
+	return append(ClassTotalSupply, []byte(classID)...)
+}
+
+// nftOfClassByOwnerStoreKey returns the byte representation of the nft owner
+func nftOfClassByOwnerStoreKey(owner sdk.AccAddress, classID string) []byte {
+	// 0x03<owner><classID><Delimiter(1 Byte)>
+
+	owner = address.MustLengthPrefix(owner)
+	classIDBz := conv.UnsafeStrToBytes(classID)
```

Comment:
I'd stick to a single str -> byte conversion, I see you are using `[]byte(classID)` in the other functions

Reply:
this is safe here, because we are only copying the content. So no need to do more memory allocation with the "safe" method.